amr-wind/physics/HybridRANSLESABL.cpp

```diff
@@ -7,6 +7,7 @@
 namespace amr_wind {
 namespace hybrid_rans_les_abl {
 
+// cppcheck-suppress uninitMemberVar
```

Comment:
is it complaining that m_tke and m_sdr are not initialized? we could set them to nullptr in the header?

Reply:
Yeah I'm pretty sure it was complaining about those. Anything we can do to not have to use suppressions is good.

Reply:
Can you guys explain your thought process? I ask because it is also complaining for smagorinsky and that class doesn't have any pointer instance members. 

Of course, defaulting the pointers for tke/SDR in this class should be done regardless of the warnings.

Reply:
While we are at it, we should also fix usages like this in Turbulence models: https://github.com/Exawind/amr-wind/blob/5525d75745b1f2a9a0a4da9504a818f8db662f77/amr-wind/turbulence/LES/Smagorinsky.cpp#L19

I think I am making a mistake here and it won't work as intended. It is better to be explicit here.

Reply:
Yeah those need to be fixed for us to get to 0 clang-tidy warnings. That's all that's left.